### DISCUSSION TITLE
Maandag 30-09-2024

### DISCUSSION THREAD

**Discussion by Nazneen05x** [rutgerkock/buurtcampus-oost#2] [Stand-ups]
* hoe gaat het?
* wat wil je deze week bereiken?
* welke taak wil je oppakken?

**Comment by Annevd** [comment]
**Hoe gaat het?:**

Het gaat tot nu toe redelijk goed. Ik heb mij afgelopen week gefocust op het testen en analyseren van de toegankelijkheid tools op de bieb in bloei website van de CMD minor. Dit is een goede basis om te beginnen aan het implementeren van de tools op de echte website van FDND. Er staan dingen in die goed werkte, maar ook dingen die nog niet goed werkte. Dit kan je terugzien in de issues.

**Wat wil je deze week bereiken?:**

Ik wil dat de nieuwe content allemaal in Hygraph klaarstaat om te gebruiken.
Ook wil ik samen met het team branches aan gaan maken waar al nodig zodat we kunnen beginnen met code.

**Welke taak wil je oppakken?:**

Ik ben van plan samen met Nazneen Hygraph volledig uit te pluizen om vervolgens de content die we van Bieb in Bloei hebben gekregen er in te zetten. Daarna wil ik wellicht een van de taken van de accessibility tools oppakken of een taak met de mascotte Harry.

**Comment by rutgerkock** [comment]
**hoe gaat het?**
Goed, ik ben uitgezocht hoe ik data van het weer uit een api kan gebruiken, en hoe ik deze aanpas. Duurde even om uite te zoeken hoe ik dit het beste kon doen, maar ik denk dat het nu goed gelukt is.
Mijn 2 taken voor deze week was het uitzoeken hoe ik verbinding maak en het weergeven van specifieke data, dit is gelukt. 

**wat wil je deze week bereiken?**
Ik wil dit komende week gaan koppelen aan een systeem waarmee ik bijv. Harry verschillende dingen kan laten doen aan de hand van deze data. Mijn doel is aan de hand van weersinformatie verschillende visuele dingen weergeven aan de hand van deze data.

**welke taak wil je oppakken?**
Dit past het beste bij issue  [#124](https://github.com/orgs/fdnd-agency/projects/3/views/1).

**Comment by wingsvn** [comment]
**Hoe gaat het?**
Het gaat tot nu toe prima. Afgelopen week had ik de taak om de accessibility tool button te ontwerpen opgepakt. Hiervoor had ik eerst nog even een analyse gedaan van andere accessibility features en buttons om te kijken wat gebruikelijk is. Uiteindelijk heb ik een aantal ontwerpen geschetst hiervoor. 

**Wat wil je deze week bereiken?**
Ik was bezig met de website testen op toegankelijkheid. Dit wil ik sowieso nog even afmaken. 

**Welke taak wil je oppakken?**
Aangezien ik bezig ben geweest met de accessibility features, wil ik dit wel verder oppakken. Eventueel wil ik ook nog kijken naar de error handling (#160).

**Comment by Lmikkers** [comment]
**Hoe gaat het?**
Het gaat goed! Ik ben begonnen met de taak om Harry te analyseren. Hiervoor heb ik het project van CMD goed bestudeerd en meerdere versies van Harry gevonden. Deze heb ik gedownload en samen met de bijbehorende "moods" in een Figma-bestand geplaatst. In de code zag ik dat ze al bezig waren om Harry te koppelen aan het weer en de temperatuur, wat we goed kunnen gebruiken voor ons eigen project. In het Figma-bestand heb ik ook kort samengevat hoe ze Harry toepassen in hun project. [Zie issue Harry analyseren](https://github.com/fdnd-agency/buurtcampus-oost/issues/122)

**Wat wil je deze week bereiken?**
Ik zou wel verder willen gaan met Harry en gaan kijken naar een manier om harry op een leuke manier te laten verschijnen, hij valt namelijk nu nog niet zo erg op. 

**Welke taak wil je oppakken?** 
Ik zou dan de taak van Harry willen oppakken en zou samen met Rutger kunnen kijken hoe we dit kunnen koppellen aan de weather api
https://github.com/fdnd-agency/buurtcampus-oost/issues/123

**Comment by Nazneen05x** [comment]
**Hoe gaat het?:**

Ik had vandaag mijn eerste mentor dag met de eerstejaars, dus een klein beetje moe. Verder gaat het goed, afgelopen twee weken bezig geweest met de agenda-carousel component.  



**Wat wil je deze week bereiken?:**

Deze week wil ik de agenda-carousel component afronden, zodat ik verder kan met andere taken.

**Welke taak wil je oppakken?:**

[#157](https://github.com/fdnd-agency/buurtcampus-oost/issues/157)
[#30](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/30)
[#22](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/22)
[#24](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/24)
[#25](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/25)
[#32](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/32)
[#29](https://github.com/Nazneen05x/S4-S19-Bieb-In-Bloei/issues/29)
